infra/gravity/node_commands.go

```diff
@@ -155,14 +155,79 @@ type ClusterStatus struct {
 	Cluster string `json:"domain"`
 	// State is the cluster state
 	State string `json:"state"`
-	// SystemStatus is the cluster status, see https://github.com/gravitational/satellite/blob/7.1.0/agent/proto/agentpb/agent.proto#L50-L54
-	SystemStatus int `json:"system_status"`
+	// SystemStatus is the cluster status, custom unmarshalling logic needed to accommodate Gravity differences
+	SystemStatus int
```

Comment:
Shouldn't then this be explicitly excluded from marshal/unmarshal?

```go
SystemStatus int `json:"-"`
```

Reply:
Good catch.  Fixed in bc97e32.